canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx

```diff
@@ -414,6 +413,7 @@ class VirtualizedTable extends React.Component {
 						readOnly={this.props.readOnly}
 					/>
 				</div>);
+			// Don't show delete icon for tables in expression builder
```

Comment:
remove this comment. This is no longer applicable now after adding delete icon in abstract-table